java/src/jmri/jmrit/roster/swing/RosterFrame.java

```diff
@@ -140,58 +140,58 @@ public RosterFrame(String name, String menubarFile, String toolbarFile) {
-    JRadioButtonMenuItem contextService = new JRadioButtonMenuItem(Bundle.getMessage("ProgrammingTrack"));
-    JTextPane dateUpdated = new JTextPane();
-    JTextPane dccAddress = new JTextPane();
-    JTextPane decoderFamily = new JTextPane();
-    JTextPane decoderModel = new JTextPane();
-    JRadioButton edit = new JRadioButton(Bundle.getMessage("EditOnly"));
-    JTextPane filename = new JTextPane();
+    final JRadioButtonMenuItem contextEdit = new JRadioButtonMenuItem(Bundle.getMessage("ButtonEdit"));
+    final JRadioButtonMenuItem contextOps = new JRadioButtonMenuItem(Bundle.getMessage("ProgrammingOnMain"));
+    final JRadioButtonMenuItem contextService = new JRadioButtonMenuItem(Bundle.getMessage("ProgrammingTrack"));
+    final JTextPane dateUpdated = new JTextPane();
+    final JTextPane dccAddress = new JTextPane();
+    final JTextPane decoderFamily = new JTextPane();
+    final JTextPane decoderModel = new JTextPane();
+    final JRadioButton edit = new JRadioButton(Bundle.getMessage("EditOnly"));
+    final JTextPane filename = new JTextPane();
     JLabel firstHelpLabel;
     //int firstTimeAddedEntry = 0x00;
     int groupSplitPaneLocation = 0;
     RosterGroupsPanel groups;
     boolean hideGroups = false;
     boolean hideRosterImage = false;
-    JTextPane id = new JTextPane();
+    final JTextPane id = new JTextPane();
     boolean inStartProgrammer = false;
     ResizableImagePanel locoImage;
     JTextPane maxSpeed = new JTextPane();
-    JTextPane mfg = new JTextPane();
-    ProgModeSelector modePanel = new ProgServiceModeComboBox();
-    JTextPane model = new JTextPane();
-    JLabel operationsModeProgrammerLabel = new JLabel();
-    JRadioButton ops = new JRadioButton(Bundle.getMessage("ProgrammingOnMain"));
+    final JTextPane mfg = new JTextPane();
+    final ProgModeSelector modePanel = new ProgServiceModeComboBox();
+    final JTextPane model = new JTextPane();
+    final JLabel operationsModeProgrammerLabel = new JLabel();
+    final JRadioButton ops = new JRadioButton(Bundle.getMessage("ProgrammingOnMain"));
     ConnectionConfig opsModeProCon = null;
-    JTextPane owner = new JTextPane();
+    final JTextPane owner = new JTextPane();
     UserPreferencesManager prefsMgr;
-    JButton prog1Button = new JButton(Bundle.getMessage("Program"));
-    JButton prog2Button = new JButton(Bundle.getMessage("BasicProgrammer"));
+    final JButton prog1Button = new JButton(Bundle.getMessage("Program"));
+    final JButton prog2Button = new JButton(Bundle.getMessage("BasicProgrammer"));
     ActionListener programModeListener;
 
     // These are the names of the programmer _files_, not what should be displayed to the user
     String programmer1 = "Comprehensive"; // NOI18N
     String programmer2 = "Basic"; // NOI18N
 
-    java.util.ResourceBundle rb = java.util.ResourceBundle.getBundle("apps.AppsBundle");
+    final java.util.ResourceBundle rb = java.util.ResourceBundle.getBundle("apps.AppsBundle");
```

Comment:
Do you intend to leave this reference to apps intact?

Reply:
We'll fix that, along with any other references to bundles, but the first goal is to make the code compile separately.

I missed it in my other PR.